css-cascade-7/Overview.bs

```diff
@@ -0,0 +1,402 @@
+	assigning its child style rules into that sheet.
+
+<h3 id="at-ruledef-import-from">
+Importing Specific Named Stylesheets: the ''from'' Keyword</h3>
+
+<pre class='prod'>
+		<dfn export>&lt;url-string></dfn> = [ <<url>> | <<string>> ]
+		<dfn export>&lt;sheet-name-list></dfn> = [ <<string>> ][ ,<<string>> ]*
+
+		@import [ <<sheet-name-list>> from ]?
+				    [ <<url-string>> ]				
+		        [ layer | layer(<<layer-name>>) ]?
+		        <<import-conditions>> ;
+
+		<dfn export>&lt;import-conditions></dfn> = [ supports( [ <<supports-condition>> | <<declaration>> ] ) ]?
+		                     <<media-query-list>>?</pre>
+
+	where:
+	* the optional <<sheet-name-list>> imports only the named sheet(s)
+		with names matching [=named stylesheet=]s from the provided URL
+		in the order they are listed.
+
+	* the <<url-string>>
+		gives the URL of the [=stylesheet=] to be imported.
+
+	* the optional ''layer'' keyword or ''layer()'' function
+		assigns the contents of the [=stylesheet=]
+		into its own anonymous [=cascade layer=]
+		or into the named [=cascade layer=].
+
+		The layer is added to the layer order
+		even if the import fails to load the stylesheet,
+		but is subject to any [=import conditions=]
+		(just as if declared by an ''@layer'' rule wrapped
+		in the appropriate [=conditional group rules=]).
+
+	* the optional <<import-conditions>>
+		states the [=import conditions=] under which it applies.
+
+	<div class=example>
+		A file with the <a href="https://www.w3.org/TR/css-cascade-5/#content-type">CSS Style Sheet Content-Type</a> 
+		and a URL of "foo.css":
+		<pre class=lang-css>
+		@sheet framework {
+			h1 { color: maroon; }
+		}
+		</pre>
+		A document with the following CSS and HTML:
+		<pre class=lang-css>
+		&lt;style>
+		@import framework from url(foo.css);
+		&lt;/style>
+		&lt;h1>Sample text&lt;/h1>
+		</pre>
+		In this example, the ''h1'' tag has the maroon color applied.
+	</div>
+
+	<div class="example">
+		The following <a href="#conditional-import">conditional <css>@import</css> rule</a>
+		only loads the [=named stylesheet=] with a name of "foo" from "narrow.css" when the UA
+		<a href="https://www.w3.org/TR/css-conditional-3/#support-definition">supports</a> ''display: flex'',
+		and only applies the [=stylesheet=] on a <a href="https://www.w3.org/TR/CSS2/media.html#media-types">handheld</a> device
+		with a <a href="https://www.w3.org/TR/mediaqueries-4/#width">maximum viewport width</a> of 400px.
+
+		<pre>@import foo from url("narrow.css") supports(display: flex) handheld and (max-width: 400px);</pre>
+	</div>
+
+	<div class="example">
+		The following layer imports the [=named stylesheet=]s named "foo" and "bar" from "tabs.css" into
+		the ''framework.component'' layer, and an un-named layer, respectively:
+
+		<pre>
+		@import foo, bar from url("tabs.css") layer(framework.component);
+		@import foo, bar from url("override.css") layer;
+		</pre>
+	</div>
+
+	If a <url-string> is provided,
```

Comment:
<url-string> isn't showing up in https://kurtcattischmidt.github.io/kurtspublishedw3cdrafts/AtSheet.html#at-ruledef-import-from. Not sure if that is just because it's an older version?

Reply:
Looks like this one is missing an extra pair of <>, so it's being rendered as a custom HTML node.
`<<url-string>>`

Reply:
Yep, that was it!